yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -240,8 +240,9 @@ describe('AVM simulator', () => {
 
         const expectedFields = [new Fr(10), new Fr(20), new Fr(30)];
         const expectedString = 'Hello, world!'.split('').map(c => new Fr(c.charCodeAt(0)));
-        // FIXME: Try this once Brillig codegen produces uniform bit sizes for LT
-        // const expectedCompressedString = Buffer.from('Hello, world!');
+        const expectedCompressedString = Buffer.from(
+          '\0A long time ago, in a galaxy fa' + '\0r far away...\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0\0',
```

Comment:
lovely padding

Reply:
Yeah man, why is the first byte the 0 one?